src/main/resources/intentionDescriptions/JoinWildcardsIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention converts multiple <code>_</code> wildcards to <code>..</code>.
```

Comment:
Maybe leave out multiple, since single `_` is now also supported?
```suggestion
This intention converts <code>_</code> wildcard(s) to <code>..</code>.
```